articles/machine-learning/v1/how-to-create-register-datasets.md

```diff
@@ -88,7 +88,10 @@ With TabularDatasets, you can specify a time stamp from a column in the data or
 Create a TabularDataset with [the Python SDK](#create-a-tabulardataset) or [Azure Machine Learning studio](../how-to-connect-data-ui.md#create-datasets).
 
 >[!NOTE]
-> [Automated ML](../concept-automated-ml.md) workflows generated via the Azure Machine Learning studio currently only support TabularDatasets. 
+> [Automated ML](../concept-automated-ml.md) workflows generated via the Azure Machine Learning studio currently only support TabularDatasets.
+> 
+>[!NOTE]
+>For TabularDatasets generating from [SQL query results](/python/api/azureml-core/azureml.data.dataset_factory.tabulardatasetfactory#from-sql-query-query--validate-true--set-column-types-none--query-timeout-30-), either T-SQL (e.g. 'WITH' sub query) or duplicate column name is not supported. Complex query like T-SQL can cause performance issues. Duplicate column names in dataset can cause ambiguity issues. 
```

Comment:
I think there are a few small grammatical things to fix here. 

It doesn't make sense to have "either" and "or" in the same sentence.

And needs to be either: "Complex quer_ies_ like T-SQL" or "A complex query like T-SQL can cause performance issues"

Last sentence: either "in datasets" or "in a dataset"